client/components/progress-bar/index.js

```diff
@@ -1 +1,5 @@
-export { ProgressBar as default } from '@automattic/calypso-ui';
+/**
+ * Internal dependencies
+ */
+import { default as ProgressBar } from '@automattic/calypso-ui/dist/esm/progress-bar';
```

Comment:
Why don't we just `import { ProgressBar } from `@automattic/calypso-ui`? The `dist/esm` directory structure should be an encapsulated implementation detail of the package, not a commonly used part of public interface.

Reply:
See p1560968242000900-slack-components

the tl;dr is that `ScreenReader` breaks SSR and `ProgressBar` breaks in tests because of `window` being undefined, if we pull in all of `calypso-ui` here (@woocommerce/components loads some dependencies that use `window`).

There is a Gutenberg PR open to guard against the usage breaking this: https://github.com/WordPress/gutenberg/pull/16227 -- so this is meant to be temporary.